src/pages/news.tsx

```diff
@@ -0,0 +1,304 @@
+  const [newsFilter, setNewsFilter] = useState<Array<NewsItemType>>([]);
+  const [filteredNews, setFilteredNews] = useState<NewsListNoDate>(news);
+
+  useEffect(
+    () =>
+      filterNews(news, newsFilter, (newNews: NewsListNoDate) =>
+        setFilteredNews(newNews)
+      ),
+    [news, newsFilter]
+  );
+
+  return (
+    <>
+      <Head>
+        <title>
+          {t("title.title")} - {t("brand.name", { ns: "common" })}
+        </title>
+      </Head>
+      <RegularLayout
+        Title={
+          <Title
+            name={{
+              title: t("title.title"),
+              // TODO: When Title line height is fixed, add this back
+              // subtitle: t("title.subtitle"),
+            }}
+            pageIcon="newspaper"
+            backGoesTo="/"
+            LinkElement={Link}
+            key="title"
+          />
+        }
+      >
+        <Section>
+          <NewsFilter setNewsFilter={setNewsFilter} />
+          <NewsMasonry news={filteredNews} />
+        </Section>
+      </RegularLayout>
+    </>
+  );
+};
+
+export const getServerSideProps: GetServerSideProps = async ({ locale }) => {
+  const news: NewsList = [
+    {
+      id: 8,
+      type: "form",
+      frequency: "once",
+      postDate: new Date(2022, 2, 20),
+      dueDate: new Date(2022, 4, 9),
+      done: false,
+      content: {
+        "en-US": {
+          title: "Student Information",
+          supportingText:
+            "Edit and confirm your student information on the Data Management Center (DMC)",
+        },
+        th: {
+          title: "ข้อมูลนักเรียนรายบุคคล",
+          supportingText: "ตรวจสอบและยืนยันข้อมูลนักเรียนรายบุคคล (DMC)",
+        },
+      },
+    },
+    {
+      id: 7,
+      type: "form",
+      frequency: "once",
+      postDate: new Date(2022, 1, 1),
+      dueDate: new Date(2022, 1, 28),
+      done: true,
+      content: {
+        "en-US": {
+          title: "Classes Feedback",
+          supportingText:
+            "All personal information will be kept as a secret. For EPlus+ students, give feedback through co-teachers.",
+        },
+        th: {
+          title: "การจัดการเรียนการสอนออนไลน์",
+          supportingText:
+            "ข้อมูลส่วนบุคคลของนักเรียนจะถูกเก็บไว้เป็นความลับ สำหรับโครงการ EPlus+ ให้ประเมินผ่าน co-teacher",
+        },
+      },
+    },
+    {
+      id: 6,
+      type: "form",
+      frequency: "weekly",
+      postDate: new Date(2022, 0, 8),
+      dueDate: new Date(2022, 2, 31),
+      done: true,
+      content: {
+        "en-US": {
+          title: "Classes Feedback",
+          supportingText:
+            "Before every Monday, all students must test for COVID-19 via RT-PCR, ATK, or other trusted methods and submit \
+            their readings here.",
+        },
+        th: {
+          title: "บันทึกผลการตรวจโควิด-19",
+          supportingText:
+            "ก่อนทุกวันจันทร์ ให้นักเรียนถ่ายรูปผลการตรวจการหาเชื้อโควิด-19 ผ่าน RT-CPR ATK หรือวิธีอื่นๆ",
```

Comment:
Not sure if this is relevant but this should be RT-PCR

Reply:
Fun fact, this was copied from News in Draft 3, which was copied from Home in Draft 3, which was copied from Home in Draft 2, which was copied from Home in Draft 1. No one noticed, until now! 🎉

Reply:
Great